src/components/player/index.js

```diff
@@ -14,7 +15,10 @@ export default class Player extends React.Component {
   }
 
   componentDidMount() {
+    // Adding keydown event listener to document element
```

Comment:
The event listener is added to `window` and not `document`.

Reply:
I've been in a dilemma whether to add the listener to document or window?